geoportal/c2cgeoportal_geoportal/scaffolds/create/docker-compose.yaml.mako

```diff
@@ -48,4 +48,13 @@ ${service_defaults('mapserver', 80)}\
 
   geoportal:
     image: ${docker_base}-geoportal:${docker_tag}
-${service_defaults('geoportal', 80, True)}\
+${service_defaults('geoportal', 80)}\
+
+  proxy:
```

Comment:
The folder is name front and the container proxy shouldn't be the same?

Reply:
I don't understand?